framework/src/security/security_utils.c

```diff
@@ -267,11 +267,14 @@ int secutils_convert_path_s2h(const char *path, uint32_t *slot)
 	if (!strncmp(path, SS_PATH, sizeof(SS_PATH) - 1)) {
 		*slot = atoi(&path[3]);
 		return 0;
-	} else if (!strncmp(path, FACTORY_PATH, sizeof(FACTORY_PATH) - 1)) {
+	}
+#if (defined(CONFIG_ARCH_BOARD_ARTIK053S) || defined(CONFIG_ARCH_BOARD_ARTIK053)) \
+		&& defined(CONFIG_S5J_HAVE_SSS)
+	else if (!strncmp(path, FACTORY_PATH, sizeof(FACTORY_PATH) - 1)) {
 		*slot = FACTORYKEY_ARTIK_CERT;
```

Comment:
This folder is a framework. Do you really want chip-specific code here??

Reply:
Oops.. then I'd rather remove this "else if" entirely.